include/earcut.hpp

```diff
@@ -124,15 +124,15 @@ void Earcut<N>::operator()(const Polygon& points) {
         do {
             x = p->x;
             y = p->y;
-            if (x < minX) minX = x;
-            if (y < minY) minY = y;
-            if (x > maxX) maxX = x;
-            if (y > maxY) maxY = y;
+            minX = (std::min)(minX, x);
```

Comment:
Well yea, but tell that Microsoft... In my own projects I usually make sure that I define NOMINMAX which disables these MIN/Max macros, but earcut is a library, and it's likely that someone else is using it in a windows project without that knowledge.